resources.go

```diff
@@ -2037,6 +2037,7 @@ func Provider() tfbridge.ProviderInfo {
 		JavaScript: &tfbridge.JavaScriptInfo{
 			Dependencies: map[string]string{
 				"@pulumi/pulumi":    "dev",
+				"aws-sdk":           "^2.0.0",
```

Comment:
Note: this means that if a person requests a different sdk version, they'll get wrong TS typings. However, that's a known limitation of the awssdk. They even call it out here: https://github.com/aws/aws-sdk-js#known-limitations

> There are a few known limitations with the bundled TypeScript definitions at this time:
> Service client typings reflect the latest apiVersion, regardless of which apiVersion is specified when creating a client.